docs/how-to/authentication-plugin-examples.md

```diff
@@ -82,36 +100,148 @@ If you're using a .NET service, refer to the [.NET Auth Core Identity plugin doc
 4. Regularly update and rotate authentication secrets and keys.
 5. Follow the principle of least privilege when assigning roles and permissions.
 
+## Disable Authentication On Your Service
+
+If you no longer need authentication on a specific service, you can disable it.
+
+1. Visit your service's Plugins page and toggle the Authentication-related plugins into the off state.
+2. Delete the User entity from your list of entities.
+3. Re-build your project and commit your changes to your preferred git provider.
+
 ## Examples
```

Comment:
I'm not sure these are examples. It looks like we have here detailed explanations for some plugins' settings.
Is it correct? If so- let's update the title and the description below